aiida/cmdline/commands/cmd_import.py

```diff
@@ -34,7 +33,55 @@ class ExtrasImportCode(Enum):
     ask = 'kca'
 
 
-def _try_import(migration_performed, file_to_import, archive, group, migration, non_interactive, **kwargs):
+def _echo_error(message, non_interactive, more_archives, progress_bar=None, debug=False, **kwargs):  # pylint: disable=unused-argument
+    """Utility function to help write an error message for ``verdi import``
+
+    :param message: Message following red-colored, bold "Error:".
+    :type message: str
+    :param non_interactive: Whether or not the user should be asked for input for any reason.
+    :type non_interactive: bool
+    :param more_archives: Whether or not there are more archives to import.
+    :type more_archives: bool
+    :param progress_bar: A tqdm progress bar instance.
+    :param debug: Whether or not debug mode was on. [Default: False]
+    :type debug: bool
+    """
+    import traceback
+
+    # Close progress bar, if it exists
+    if progress_bar:
+        if not debug:
+            progress_bar.leave = False
+        progress_bar.close()
+
+    if debug:
+        exception = traceback.format_exc()
+    else:
+        exception = traceback.format_exc(limit=0).splitlines()[-1]
```

Comment:
I think what you do below can simply be handled by something like `str(e)` or `e.msg` where `e` is the `Exception` instance, instead of parsing from the traceback.
You may need to pass the exception to the `_echo_error` function explicitly, which is perhaps anyhow a bit cleaner.

Reply:
I think you're right. My original reason, if I remember correctly, for doing this was that to limit the traceback stack to the immediate exception thrown - for a cleaner output.